src/components/event/details/DetailsSuspectedCommits.vue

```diff
@@ -0,0 +1,180 @@
+    </template>
+    <template #content>
+      <div
+        v-for="(commit, index) in filteredCommits"
+        :key="index"
+        class="
+          event-details__content-block
+          details-suspected-commit__content-block
+        "
+        data-ripple
+      >
+        <div class="details-suspected-commit__left">
+          <div class="details-suspected-commit__left-title">
+            {{ commit.title }}
+          </div>
+          <div>
+            <EntityImage
+              class="details-suspected-commit__left-image"
+              :name="commit.author"
+              size="16"
+            />
+            <span class="details-suspected-commit__left-author">
+              {{ commit.author }}
+            </span>
+            <span class="details-suspected-commit__left-relative-time">
+              {{ $t("event.suspectedCommits.committed") }}
+              {{ commit.date | prettyRelativeTimeStr }}
+            </span>
+          </div>
+        </div>
+        <div class="details-suspected-commit__right">
+          <div class="details-suspected-commit__right-hash-block">
+            <span class="details-suspected-commit__right-hash">
+              {{ commit.hash.substr(0, 7) }}
+            </span>
+          </div>
+        </div>
+      </div>
+    </template>
+    <template #expandButton>
+      {{
+        isMoreCommitsShown
+          ? $t("event.suspectedCommits.hide")
+          : $tc(
+            "event.suspectedCommits.moreRelease",
+            commits.length - numberOfVisibleCommits,
+            {
+              numberOfCommits: commits.length - numberOfVisibleCommits,
+            }
+          )
+      }}
+    </template>
+  </DetailsBase>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import DetailsBase from './DetailsBase.vue';
+import EntityImage from '../../utils/EntityImage.vue';
+import { HawkEventCommit } from '@/types/events';
+
+export default Vue.extend({
+  name: 'DetailsSuspectedCommits',
+  components: {
+    DetailsBase,
+    EntityImage
+  },
+  props: {
+    /**
+     * Event commits to show
+     */
+    commits: {
+      type: Array as () => HawkEventCommit[],
+      required: true,
+    },
+    
+  },
+  data() {
+    return {
+      /**
+       * Is block expanded.
+       */
+      isMoreCommitsShown: false,
+      /**
+       * Number of Visible Commits
+       */
+      numberOfVisibleCommits : 1,
+    
+    };
+  },
+  computed: {
+    /**
+     * Displayed commits items
+     * @returns { HawkEventCommit[] }
+     */
+    filteredCommits():HawkEventCommit[] {
+      return this.commits.length <= this.numberOfVisibleCommits ||
+        this.isMoreCommitsShown
+        ? this.commits
+        : this.commits.slice(0, this.numberOfVisibleCommits);
+    },
+  },
+});
+</script>
+
+<style>
+.details-suspected-commit {
+  &__content-block {
+    display: flex;
+    flex-direction: row;
+    padding: 13px 15px;
+  }
+  &__left > div {
+    height: 16px;
```

Comment:
you should not hardcode a height of an element, It can be configured using `font-size`,  `line-heigth`,  and `padding`